hie-plugin-api/Haskell/Ide/Engine/Config.hs

```diff
@@ -36,6 +37,7 @@ instance Default Config where
     , liquidOn                    = False
     , completionSnippetsOn        = True
     , formatOnImportOn            = True
+    , formattingProvider          = "brittany"
```

Comment:
Could this be a datatype instead of some text?

Reply:
I actually originally had this as a datatype! But when adding the `pluginFormattingProvider` field to the plugin descriptor it made more sense to revert it back to just the `PluginId`, that way if a new plugin is added the developer doesn't need to edit `Config.hs` as well. 
I do agree though, if there are any tricks to having the `PluginId`s exported in a more type safe manner then I would much prefer that